config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
- [ ] I will encourage you to use a more expressive and concise way, with the one you have used, I am not sure it would work because you did not add the `to`. However, I will encourage you to set it as the following:
```
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

With that, it would be concise and readable.